stdlib/SparseArrays/src/sparsematrix.jl

```diff
@@ -531,16 +531,14 @@ function SparseMatrixCSC{Tv,Ti}(M::StridedMatrix) where {Tv,Ti}
 end
 SparseMatrixCSC(M::Adjoint{<:Any,<:SparseMatrixCSC}) = copy(M)
 SparseMatrixCSC(M::Transpose{<:Any,<:SparseMatrixCSC}) = copy(M)
-SparseMatrixCSC{Tv}(M::Adjoint{Tv,SparseMatrixCSC{Tv}}) where {Tv} = copy(M)
```

Comment:
Are you sure these aren't needed? Don't they avoid a copy if the element type is correct?

Reply:
Seems like they are used and tested:
https://coveralls.io/builds/29001780/source?filename=stdlib/SparseArrays/src/sparsematrix.jl#L584